File: My Practice Projects/PracticeDialog/app/src/main/java/com/example/farhan/practicedialog/MainActivity.java
package com.example.farhan.practicedialog;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btn = (Button) findViewById(R.id.mbtn);

        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                CustomDailog customDailog = new CustomDailog();
                customDailog.show(getFragmentManager(),"asd");
            }
        });
    }
}

File: My Practice Projects/PracticeKotlinRecyclearView/app/src/main/java/com/example/farhan/practicekotlinrecyclearview/MainActivity.kt
package com.example.farhan.practicekotlinrecyclearview

import android.os.Bundle
import android.support.v7.app.AlertDialog
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView
import android.widget.Button
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    lateinit var customRCView: CustomAdapter
    lateinit var data: ArrayList<String>
    lateinit var recyclerView: RecyclerView


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.reservation_form_b)

        recyclerView = findViewById(R.id.form_b_RV)
        data = arrayListOf()

        customRCView = CustomAdapter(data)

        data.add("test")
        data.add("test")
        data.add("test")

        recyclerView.layoutManager = LinearLayoutManager(this)
        recyclerView.adapter = customRCView


        /*btn_Show.setOnClickListener {
            val builder = AlertDialog.Builder(this)
            val view = layoutInflater.inflate(R.layout.reservation_dialog_view, null)

            builder.setView(view)
            builder.create().show()

        }*/

    }
}
